src/pages/ContentScripts/features/repo-activity-openrank-trends/index.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { render, Container } from 'react-dom';
+import elementReady from 'element-ready';
+import $ from 'jquery';
+import * as pageDetect from 'github-url-detection';
+
+import features from '../../../../feature-manager';
+import { getRepoName } from '../../../../helpers/get-repo-info';
+import { getActivity, getOpenrank } from '../../../../api/repo';
+import View from './view';
+
+const featureId = features.getFeatureID(import.meta.url);
+let repoName: string;
+let activity: any;
+let openrank: any;
+
+const getData = async () => {
+  try {
+    activity = await getActivity(repoName);
+    openrank = await getOpenrank(repoName);
+  } catch (e) {
+    console.error(e);
+  }
+};
+
+const renderTo = (container: Container) => {
+  render(
+    <View repoName={repoName} activity={activity} openrank={openrank} />,
+    container
+  );
+};
+
+const init = async (): Promise<void> => {
+  repoName = getRepoName();
+  await getData();
+
+  // create container
+  const newBorderGridRow = document.createElement('div');
+  newBorderGridRow.id = featureId;
+  newBorderGridRow.className = 'BorderGrid-row';
+  const newBorderGridCell = document.createElement('div');
+  newBorderGridCell.className = 'BorderGrid-cell';
+  newBorderGridRow.appendChild(newBorderGridCell);
+
+  // we don't wait all DOM, but the anchor element should be waited.
```

Comment:
This comment could be confusing. Did you mean we don't need to wait until all DOM are ready, but only the anchor element? BTW, is the `anchor element` a term for this situation? feels like `parent element` is more intuitive to me.

Reply:
> Did you mean we don't need to wait until all DOM are ready, but only its anchor element?

You got me! I wanted to make the comment short and compact, but apparently I didn't  express myself clearly.

Will improve it.